src/aiidalab_qe/plugins/xps/setting.py

```diff
@@ -0,0 +1,249 @@
+        structures. The default value of 8.0 angstrom will be used
+        if no input is given. Setting this value to 0.0 will
+        instruct the CF to not scale up the input structure.
+        </div>"""
+    )
+    binding_energy_title = ipw.HTML(
+        """<div style="padding-top: 0px; padding-bottom: 0px">
+        <h4>Absolute binding energy</h4></div>"""
+    )
+    binding_energy_help = ipw.HTML(
+        """<div style="line-height: 140%; padding-top: 10px; padding-bottom: 10px">
+        To calculate the absolute binding energy, you need to provide the correction energy for the core electrons. The correction energy is Ecorr = E_core_hole - E_gipaw, where E_core_hole and E_gipaw are calculated by Etot - Etotps. Etot and Etotps can be found in the output when generating the pseudo potential. A offset corretion by fitting the experimental data is also added. Here is a example: C:339.79,O:668.22,F:955.73,Si:153.19
+        </div>"""
+    )
+
+    def __init__(self, **kwargs):
+        # Core hole treatment type
+        self.core_hole_treatment = ipw.ToggleButtons(
+            options=[
+                ("XCH(smear)", "xch_smear"),
+                ("XCH(fixed)", "xch_fixed"),
+                ("Full", "full"),
+            ],
+            value="xch_smear",
+        )
+        self.pseudo_group = ipw.Dropdown(
+            options=["pseudo_demo_pbe", "pseudo_demo_pbesol"],
+            value="pseudo_demo_pbe",
+            description="Group:",
+            disabled=False,
+            style={"description_width": "initial"},
+        )
+        self.peak_list = ipw.VBox()
+
+        self.structure_type = ipw.ToggleButtons(
+            options=[
+                ("Molecule", "molecule"),
+                ("Crystal", "crystal"),
+            ],
+            value="crystal",
+        )
+        self.supercell_min_parameter = ipw.FloatText(
+            value=8.0,
+            description="The minimum cell length (Å):",
+            disabled=False,
+            style={"description_width": "initial"},
+        )
+        self.calc_binding_energy = ipw.Checkbox(
+            description="Calculate binding energy: ",
+            indent=False,
+            value=False,
+        )
+
+        self.children = [
+            self.structure_title,
+            self.structure_help,
+            ipw.HBox(
+                [self.structure_type],
+            ),
+            # self.core_hole_treatment_title,
+            # self.core_hole_treatment_help,
+            # self.core_hole_treatment,
+            self.pseudo_title,
+            self.pseudo_help,
+            self.pseudo_group,
+            self.peak_title,
+            self.peak_help,
+            ipw.HBox(
+                [self.peak_list],
+            ),
+            # self.supercell_title,
+            # self.supercell_help,
+            # ipw.HBox(
+            # [self.supercell_min_parameter],
+            # ),
```

Comment:
There are quite a few commented-out sections here which should be cleaned up if possible:
```suggestion
            self.pseudo_title,
            self.pseudo_help,
            self.pseudo_group,
            self.peak_title,
            self.peak_help,
            ipw.HBox(
                [self.peak_list],
            ),
```

Does this also mean (based on the fact that `supercell_min_parameter` is not used) that the supercell size used in calculation is defined entirely by the protocol?

Reply:
Yes. This would be more user friendly